cfgov/paying_for_college/migrations/0001_initial.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-06-14 21:00
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Alias',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('alias', models.TextField()),
+                ('is_primary', models.BooleanField(default=False)),
+            ],
+            options={
+                'verbose_name_plural': 'Aliases',
+            },
+        ),
+        migrations.CreateModel(
+            name='BAHRate',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('zip5', models.CharField(max_length=5)),
+                ('value', models.IntegerField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='ConstantCap',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('slug', models.CharField(blank=True, help_text='VARIABLE NAME FOR JS', max_length=255)),
+                ('value', models.IntegerField()),
+                ('note', models.TextField(blank=True)),
+                ('updated', models.DateField(auto_now=True)),
+            ],
+            options={
+                'ordering': ['slug'],
+            },
+        ),
+        migrations.CreateModel(
+            name='ConstantRate',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255)),
+                ('slug', models.CharField(blank=True, help_text='VARIABLE NAME FOR JS', max_length=255)),
+                ('value', models.DecimalField(decimal_places=5, max_digits=6)),
+                ('note', models.TextField(blank=True)),
+                ('updated', models.DateField(auto_now=True)),
+            ],
+            options={
+                'ordering': ['slug'],
+            },
+        ),
+        migrations.CreateModel(
+            name='Contact',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('contacts', models.TextField(blank=True, help_text='COMMA-SEPARATED LIST OF EMAILS')),
+                ('endpoint', models.TextField(blank=True)),
```

Comment:
The equivalent migration in college-costs [here](https://github.com/cfpb/college-costs/blob/a75769f0df9d1ceabaf991cf83e4dba8b48c308b/paying_for_college/migrations/0001_initial.py#L65) has a limit of `max_length=255` for the `endpoint` and `name` fields because of the definitions [here](https://github.com/cfpb/college-costs/blob/54004216e4dd0fc2c460858b1e2689eb29fd9676/paying_for_college/models.py#L179-L180), which is changed in this PR [here](https://github.com/cfpb/cfgov-refresh/pull/5065/files#diff-7463f0a6105cc526d69708a8635bc723R179). Was this a deliberate change?

This will cause a problem if data greater than 255 characters is ever entered into these fields, because the database table is defined with that limit:

```sql
cfgov=> \d paying_for_college_contact;
                                        Table "cfpb.paying_for_college_contact"
    Column     |          Type          | Collation | Nullable |                        Default                         
---------------+------------------------+-----------+----------+--------------------------------------------------------
 id            | bigint                 |           | not null | nextval('paying_for_college_contact_id_seq'::regclass)
 endpoint      | character varying(255) |           | not null | 
 name          | character varying(255) |           | not null | 
 internal_note | text                   |           | not null | 
 contacts      | text                   |           | not null | 
Indexes:
    "idx_16899_primary" PRIMARY KEY, btree (id)
```

If we want to change these fields to remove the length restriction, there needs to be another Django migration to make the DB schema change.

Reply:
It was a deliberate change ... when I was planning to create new tables. As there is no longer an advantage to specifying varchar(255) in postgres, I was going to leave that behind, but as we're reusing the pfc tables, I should leave the definition as-is. Thanks for catching this.